Source/SwiftLintFramework/Extensions/String+SwiftLint.swift

```diff
@@ -23,11 +23,11 @@ extension String {
     }
 
     internal func isUppercase() -> Bool {
-        return self == uppercased()
+        return CharacterSet.uppercaseLetters.isSuperset(ofCharactersIn: self)
     }
 
     internal func isLowercase() -> Bool {
-        return self == lowercased()
+        return CharacterSet.lowercaseLetters.isSuperset(ofCharactersIn: self)
```

Comment:
If it really means, “is lowercase” then you need the following (see explanation above):

```swift
return unicodeScalars.contains(where: CharacterSet.lowercaseLetters.contains) // Ll
```

...but that leaves many characters that are neither. Simply using `!isUppercase` is probably what you actually want to be using instead.

(It does not look like this property is used by any other rule, so if you make the adjustments listed below, it can be dropped entirely.)